### PR TITLE
New module to add timeout for fetchmail servers

### PR DESCRIPTION
**Pull request by mdietrichc2c**
This PR is to add what was done in https://github.com/OCA/OCB/pull/292 in a specific module.

### CODE REVIEW

File: mail_timeout/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+#
+##############################################################################
+
+{
+    'name': 'Mail timeout',
+    'version': '0.1',
+    'category': 'Tools',
+    'author': "Camptocamp,Odoo Community Association (OCA)",
+    'summary': 'Clean up mails regularly',
+    'description': """
```
Comment (lepistone) [inline mail_timeout/__openerp__.py:28]: Not needed: we have a `README`

Comment (pedrobaeza) [inline mail_timeout/__openerp__.py:28]: This is v7, so unfortunately it's needed

**Comment by pedrobaeza** [comment]
:+1:

I insist that maybe this should be treated as a bug and put in OCB a fixed timeout as workaround

**Comment by gurneyalex** [comment]
@pedrobaeza sorry I was not fully aware of the issue and the initial PR or of the problem. 

If this is considered a bug, then we can probably have a hard coded timeout in the code without adding a field, indeed.

**Comment by mdietrichc2c** [comment]
@pedrobaeza @gurneyalex I re-re-did this in OCB/odoo: https://github.com/odoo/odoo/pull/7790 and https://github.com/OCA/OCB/pull/294

**Comment by dreispt** [comment]
If a parameters is needed, you can add it to `ir.config_parameter` instead of having a hardcoded value.

**Comment by dreispt** [comment]
Remember that for OCB a fix PR to upstream Odoo is required.

**Pull request closed without merging**